segmentstore/server/host/src/test/java/io/pravega/segmentstore/server/host/StorageLoaderTest.java

```diff
@@ -49,16 +61,88 @@ public void testNoOpWithWithInMemoryStorage() throws Exception {
 
         configBuilder
                 .include(StorageExtraConfig.builder()
-                .with(StorageExtraConfig.STORAGE_NO_OP_MODE, false));
+                        .with(StorageExtraConfig.STORAGE_NO_OP_MODE, false));
 
         builder = ServiceBuilder.newInMemoryBuilder(configBuilder.build())
                 .withStorageFactory(setup -> {
                     StorageLoader loader = new StorageLoader();
-                    expectedFactory = loader.load(setup, "INMEMORY", executor);
+                    expectedFactory = loader.load(setup, "INMEMORY", StorageLayoutType.ROLLING_STORAGE, StorageMetadataFormat.HEADER_BASED, executor);
                     return expectedFactory;
                 });
         builder.initialize();
         assertTrue(expectedFactory instanceof InMemoryStorageFactory);
         builder.close();
     }
+
+    @Test
+    public void testFileSystemStorage() throws Exception {
+        val storageType = ServiceConfig.StorageType.FILESYSTEM;
+        ServiceBuilder builder = getStorageFactory(storageType, "FILESYSTEM", StorageLayoutType.ROLLING_STORAGE, StorageMetadataFormat.HEADER_BASED);
+        assertTrue(expectedFactory instanceof FileSystemStorageFactory);
+        builder.close();
+    }
+
+    @Ignore
```

Comment:
Why is this being ignored? I actually see a few test cases ignored in this class.

Reply:
The tests need to be fixed.